lib/src/rules/public_member_api_docs.dart

```diff
@@ -226,11 +226,21 @@ class _Visitor extends SimpleAstVisitor {
 
   @override
   void visitConstructorDeclaration(ConstructorDeclaration node) {
-    if (!inPrivateMember(node) &&
-        !isPrivate(node.name) &&
-        node.parent is! EnumDeclaration) {
-      check(node);
+    if (inPrivateMember(node) || isPrivate(node.name)) return;
```

Comment:
The code duplication with an earlier PR concerns me. I'm wondering whether we want to add the lines below to `inPrivateMember` (or add a new utility method to include the overall semantics), maybe making it an extension method in the process.

Reply:
Agreed. I've got another PR in the wings for `@internal`. I'll address this duplication there. Thanks!